cli/pkg/kctrl/cmd/package/repository/release/artifact_writer.go

```diff
@@ -31,6 +31,9 @@ func (w *ArtifactWriter) WritePackageRepositoryFile(imgpkgBundleLocation string)
 		ObjectMeta: metav1.ObjectMeta{
 			Name:              w.PackageRepoName,
 			CreationTimestamp: metav1.NewTime(time.Now()),
+			Annotations: map[string]string{
+				"tag": tag,
```

Comment:
Let's call the annotation `kctrl.carvel.dev/repository-version`.

Let's save this in a constant and rescue it when we are looking for the tag

Reply:
Adopted